portal/server/types/view-models/record-correction/provide-utilisation-report-correction.ts

```diff
@@ -0,0 +1,16 @@
+import { CurrencyAndAmountString, RecordCorrectionReason } from '@ukef/dtfs2-common';
+import { PrimaryNavKey } from '../../primary-nav-key';
+
+export type CorrectionRequestDetailsViewModel = {
+  facilityId: string;
+  exporter: string;
+  reportedFees: CurrencyAndAmountString;
+  reasons: RecordCorrectionReason[];
+  formattedReasons: string;
```

Comment:
Note that we need both of these:
- `reasons` so we can selectively render the form fields (subsequent PR)
- `formattedReasons` to render a friendly string in the 'Current record' section